Bug was reported to trac:
https://trac.videolan.org/vlc/ticket/16768

This patch was suggested by courmisch in the trac ticket.

Signed-off-by: Bernd Kuhls <bernd.kuhls <at> t-online.de>
---
 include/vlc/libvlc_media.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git include/vlc/libvlc_media.h b/include/vlc/libvlc_media.h
index 3e7c637..7caa65e 100644
--- include/vlc/libvlc_media.h
+++ include/vlc/libvlc_media.h
 <at>  <at>  -26,6 +26,8  <at>  <at> 
 #ifndef VLC_LIBVLC_MEDIA_H
 #define VLC_LIBVLC_MEDIA_H 1

+#include <sys/types.h>  /* for ssize_t */
+
 # ifdef __cplusplus
 extern "C" {
 # endif
--
